pandas/tests/indexes/datetimes/test_construction.py

```diff
@@ -26,25 +27,28 @@ def test_construction_caching(self):
                                                  freq='ns')})
         assert df.dttz.dtype.tz.zone == 'US/Eastern'
 
-    def test_construction_with_alt(self):
-
+    @pytest.mark.parametrize('kwargs', [
```

Comment:
can you parameterize on tz_aware_fixture (doesn't include None), then parameterize as you are doing